tsconfig.schematics.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I'm not sure how this lib is release, but shouldn't the output but in the `dist` folder?

Reply:
We write to `dist` folder